test/test_retriever.py

```diff
@@ -105,7 +105,7 @@ def test_auto_get_datatypes():
     The function adds 100 to the auto detected length of column
     """
     test_engine.auto_get_datatypes(None,
-                                   [["�", 'bb', 'L�ve']],
+                                   [["�", 'bb', 'L�ve']],
```

Comment:
@Jaffrey98, Why are these characters changing?

Reply:
Ummm. Not sure. I'll check n revert back.